### PR TITLE
Alerts

### PR DESCRIPTION
**Pull request by aatishpandey**
image
Alert component includes:
- Primary Alert
- Secondary Alert
- Success Alert
- Danger Alert
- Warning Alert

### CODE REVIEW

**Review by aishwarya-bm — APPROVED** [review]
looks good

File: components/component-stylesheets/alerts.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+.alert{
+    font-size: 1.5rem;
+    padding: 1rem 1rem;
```
Comment (RahulSingh9131) [inline components/component-stylesheets/alerts.css]: you can use the shorthand property here, if you want to give 1rem to top-bottom & left-right.

**Review by RahulSingh9131 — COMMENTED** [review]
everything looks good to me .

File: components/component-stylesheets/alerts.css (hunk 1 above)
Comment (aatishpandey) [reply components/component-stylesheets/alerts.css]: thanks for pointing it out. fixed in recent commit.

**Review by RahulSingh9131 — APPROVED** [review]

**Pull request merged**